### DISCUSSION TITLE
Set direction of skins - top and bottom?

### DISCUSSION THREAD

**Discussion by Ltek** [OrcaSlicer/OrcaSlicer#1491] [General]
I have a part where I want both skins (top-most and bottom-most / on the plate) to be the same direction (90 degrees).  Changing infill direction changes the top-most skin but nothing I can find changes the bottom?

tia!

**Comment by Ltek** [comment]
bueler

**Comment by KD7LRJ** [comment]
I didn't understand your message when I first read it and still don't after trying again. If both surfaces (top and bottom) are to "be the same direction," what does the "(90 degrees)" part mean? Would rotating the part in the Z axis not accomplish what you are after? Pictures with notes may help (for me, anyway).

**Reply by Ltek** [reply]
screens attached. I want both top & bottom direction to extrude the same direction. according to the docs, that is what the setting "Infill direction" is supposed to set. So either its broken, or that is not what that setting does. Rotating the part (or changing the setting "Infill direction" doesnt fix making them the same direction, it just changes which side is one way or the other, they are never both the same
![Merged_document](https://github.com/SoftFever/OrcaSlicer/assets/1626802/4c3667bf-d333-420b-b8a4-f0d12218363c)

**Reply by KD7LRJ** [reply]
Seems like this is just a matter of which direction the layers happen to run at the height where the top layer of the object ends up. I would try changing either the height of the object itself (if it's not critical) or the layer height by a tiny amount so you end up with either one less or one more layer that runs in the preferred direction.

**Reply by Ltek** [reply]
My ask was hoping a dev or expert/vet would answer
a - If this is a bug or this setting is incorrectly documented
and/or
b - if there is a proper setting for this like there is in Cura.

There really needs to be 3 separate direction settings... infill, shell top, shell bottom
